contracts/button/errors.rs

```diff
@@ -21,6 +25,10 @@ pub enum GameError {
     CantRetrieveOwnCodeHash,
     /// Overflow error
     Arithmethic,
+    /// Error from the marketplace contract
+    MarketplaceError(MarketplaceError),
+    /// Error while calling another contract
+    ContractCall(LangError),
```

Comment:
Just FYI, This is removed in the next release of ink (release candidate) as `LangError`s are not expected to be recoverable so the defaults will be to panic - https://github.com/paritytech/ink/pull/1512

Reply:
Yeah, I know, we'll deal with that if/when we migrate to that release.